integration_tests/e2e/health.cy.ts

```diff
@@ -27,4 +27,13 @@ context('Healthcheck', () => {
       })
     })
   })
+
+  context('Sentry check', () => {
+    it('Throws a test error', () => {
+      cy.request({ url: '/debug-sentry', method: 'GET', failOnStatusCode: false }).then(response => {
+        expect(response.status).to.equal(500)
+        expect(response.body).to.contain('Test Sentry error thrown.')
```

Comment:
Nice!

Would it be worth including a test here to demonstrate that the PII are being anonymised as expected?

As far as I can tell we aren't actually able to run this test at present? I've seen [this PR](https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-ui/compare/220-e2e-tests) about using Playwright but I think that is a work in progress?

Reply:
> As far as I can tell we aren't actually able to run this test at present?

ah yeah that's a good point 🤦 I've realised this is in the `e2e` folder, but I think it can be run as a regular integration test...

> Would it be worth including a test here to demonstrate that the PII are being anonymised as expected?

That's a really good idea!

Reply:
ah just actually looking at the context of this test, because I realised it wasn't really testing anything... it looks like it was put in to force a production error to be thrown in order for the environment to be seen https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/commit/2acadacbbab80a498c6675b3de905011a4b7d6d1